CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 * Minimum supported Java version is now 8 (was 7).
 * Updated `circus-train-version` to 16.0.0 (was 15.0.0).
 * Updated `hotels-oss-parent` version to 5.0.0 (was 4.2.0).
-* Updated `hive` version to 2.3.5 (was 2.3.3).
+* Updated `hive` version to 2.3.7 (was 2.3.3).
```

Comment:
```suggestion
* Updated `hive` version to 2.3.7 (was 2.3.3). This allows Circus Train Big Query to be used on JDK>=9.
```

Reply:
Your CHANGELOG message says old version was `2.3.3` but the change in the POM says the old version was `2.3.5`.